api/posts/create.ts

```
@@ -46,6 +46,7 @@ const create = async (props: CreatePostProps) => {
     });
   } catch (error) {
     console.error("Error adding item:", error);
+    throw error;
```

tried with local backend/turning it off, looks it catches the error here and then continues execution, and goes to the success even when the post creation fails. instead we need to propagate the error by throwing the error from the backend so it'll get caught in the try-catch block of uploadAllImages

---

ohh good catch